developer/hosting/creating-a-constrained-runspace.md

```diff
@@ -103,7 +103,7 @@ namespace Microsoft.Samples.PowerShell.Runspaces
 
 ## Making commands private
 
- You can also make a command private, by setting it's [System.Management.Automation.Commandinfo.Visibility*](/dotnet/api/System.Management.Automation.CommandInfo.Visibility) property to [System.Management.Automation.Sessionstateentryvisibility.Private](/dotnet/api/System.Management.Automation.SessionStateEntryVisibility.Private). The host application and other commands can call that command, but the user of the application cannot. In the following example, the [Get-ChildItem](/powershell/module/Microsoft.PowerShell.Management/Get-ChildItem) command is private.
+ You can also make a command private, by setting it's [System.Management.Automation.Commandinfo.Visibility*](/dotnet/api/System.Management.Automation.CommandInfo.Visibility) property to [System.Management.Automation.Sessionstateentryvisibility.Private](/dotnet/api/system.management.automation.sessionstateentryvisibility). The host application and other commands can call that command, but the user of the application cannot. In the following example, the [Get-ChildItem](/powershell/module/Microsoft.PowerShell.Management/Get-ChildItem) command is private.
```

Comment:
The link text should match the target. Also need to remove the `*` character from the link texts.

```markdown
[System.Management.Automation.Commandinfo.Visibility*](/dotnet/api/System.Management.Automation.CommandInfo.Visibility) property to **Private**.
```